modules/i18n/src/translations/en-US/portals/console.ts

```diff
@@ -7479,7 +7479,8 @@ export const console: ConsoleNS = {
                         subtitles: {
                             0: "There are no organizations at the moment.",
                             1: "You can add a new organization easily by",
-                            2: "clicking on the button below."
```

Comment:
Shall we make this change in the French and Sinhalese files as well?

Reply:
Added